apps/web/src/components/auth/RegisterForm.tsx

```diff
@@ -17,203 +17,109 @@ import { Form } from 'ui';
 };
 
 export const RegisterForm = ({ onSuccess }: IRegisterFormProps) => {
-  const register = useRegister();
-  const [data] = useState([
+  const [data, setData] = useState([
     { value: 'consumer', label: 'Consumer' },
     { value: 'merchant', label: 'Merchant' },
   ]);
 
-  const handleOnSubmit = (values) => {
-    values.address.zipcode = parseInt(values.address.zipcode, 10);
-    register.mutate(values, {
-      onSuccess: () => {
-        onSuccess();
-      },
-    });
-  };
-
   return (
     <StyledContainer>
       <Paper radius="md" p="xl" withBorder>
         <Form<TRegisterValues, typeof schema>
-          onSubmit={(values) => handleOnSubmit(values)}
+          onSubmit={async () => {
+            onSuccess();
+          }}
           schema={schema}
         >
-          {({ formState, register, setValue }) => (
+          {({ formState }) => (
             <>
               <Stack>
                 <Title order={1}>Register</Title>
                 <Select
                   label="User Type"
                   data={data}
-                  placeholder="Select User Type"
-                  required
-                  onChange={(selectedValue: string) => {
-                    setValue('userType', selectedValue);
-                  }}
-                  error={formState.errors['userType']?.message}
+                  placeholder="Select items"
+                  nothingFound="Nothing found"
+                  searchable
                 />
                 <TextInput
                   label="User Name"
                   placeholder="Enter your User Name..."
                   required
-                  error={formState.errors['username']?.message}
-                  {...register('username')}
                 />
                 <Group grow>
                   <TextInput
                     label="First Name"
                     placeholder="Enter your First Name"
                     required
-                    error={formState.errors['firstName']?.message}
-                    {...register('firstName')}
                   />
                   <TextInput
                     label="Last Name"
                     placeholder="Enter your Last Name"
                     required
-                    error={formState.errors['lastName']?.message}
-                    {...register('lastName')}
                   />
                 </Group>
                 <TextInput
                   label="Email"
                   placeholder="Enter your Email"
                   required
-                  error={formState.errors['email']?.message}
-                  {...register('email')}
-                />
-                <TextInput
-                  label="Phone Number"
-                  placeholder="Enter your Number"
-                  required
-                  error={formState.errors['phoneNumber']?.message}
-                  {...register('phoneNumber')}
                 />
                 <TextInput
```

Comment:
```suggestion
                <TextInput
                  label="Address"
                  placeholder="House No. / Street / Barangay / City / Province"
                  value={inputValues.address}
                  onChange={(event) => handleInputChange(event, 'address')}
                  onBlur={() => handleInputBlur('address')}
                  required
                />
```

then use the function like this for the TextInput